python/hail/expr/functions.py

```diff
@@ -1710,23 +1711,23 @@ def rand_norm(mean=0, sd=1, seed=None) -> Float64Expression:
 
     Parameters
     ----------
-    mean : float or :class:`.Expression` of type :py:data:`.tfloat64`
+    mean : :obj:`float` or :class:`.Float64Expression`
         Mean of normal distribution.
     sd : float or :class:`.Expression` of type :py:data:`.tfloat64`
         Standard deviation of normal distribution.
-    seed : :obj:`int` or `None`
-        If not `None`, function will be seeded with provided seed.
+    seed : :obj:`int`, optional
+        Random seed.
 
     Returns
     -------
-    :class:`.Expression` of type :py:data:`.tfloat64`
+    :class:`.Float64Expression`
     """
     return _seeded_func("rand_norm", tfloat64, seed, mean, sd)
 
 
 @typecheck(lamb=expr_float64, seed=nullable(int))
 def rand_pois(lamb, seed=None) -> Float64Expression:
-    """Samples from a Poisson distribution with rate parameter `lamb` (RNG).
+    """Samples from a Poisson distribution with rate parameter `lamb`.
```

Comment:
please change parameter name to lambda and link to https://en.wikipedia.org/wiki/Poisson_distribution

Reply:
`lambda` is a reserved word in python, unfortunately